ada/language/ast.py

```diff
@@ -1968,14 +1968,14 @@ def constrain_prefix(prefix=T.Expr):
         return LogicTrue()
 
     relative_name = Property(
-        Entity.defining_name.relative_name, public=True, doc="""
+        Entity.defining_name._.relative_name, public=True, doc="""
         Return the relative name for Self. If Self's defining name is
         ``A.B.C``, return C as a node.
         """
     )
 
     relative_name_text = Property(
-        Entity.relative_name.symbol, doc="""
+        Entity.relative_name._.symbol, doc="""
```

Comment:
Please add a testcase.

Also, shouldn’t the commit message be “make p_relative_name return null when there is no defining_name”? (here it sounds like “make it aways return null”)

Thanks!